spec.html

```diff
@@ -35474,7 +35474,7 @@ <h1>%TypedArray%.of ( ..._items_ )</h1>
 
       <emu-clause id="sec-%typedarray%.prototype">
         <h1>%TypedArray%.prototype</h1>
-        <p>The initial value of %TypedArray%`.prototype` is the %TypedArray.prototype% intrinsic object.</p>
+        <p>The initial value of %TypedArray%`.prototype` is the %TypedArray% prototype object.</p>
```

Comment:
I'm not sure about the backtick placement here. But since this doesn't change it, we can leave it alone.